conda/exceptions.py

```diff
@@ -486,6 +481,36 @@ def __init__(self, message):
         super(CondaVerificationError, self).__init__(message)
 
 
+class CondaDependencyError(CondaError):
+    def __init__(self, message):
+        super(CondaDependencyError, self).__init__(message)
+
+
+class PrefixReplacementError(CondaError):
+    def __init__(self, message, new_data_length, original_data_length, new_prefix,
```

Comment:
Don't need the first `message` argument here.  Need it in the `super()` call of course.